File: backend/src/main/java/com/gnome/crossing/backend/model/ItemLocation.java
package com.gnome.crossing.backend.model;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author davidjmartin
 */
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class ItemLocation {

    private Long id;

    private String imageUrl;
    private String description;
    private String location;

    private List<Keyword> locationKeywords;

}

File: frontend/src/app/_models/ItemLocation.ts
import { Keyword } from "./Keyword";

export class ItemLocation {

    constructor(uuid: string, imageUrl: any, description: any) {        
        this.uuid = uuid;
        this.imageUrl = imageUrl;
        this.description = description;
    }
    
    id!: number;
    uuid!:string;

    imageUrl : string = '';
    description : string = '';
    location : string = '';

    // locationKeywords: Keyword[];

}
